gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/DisplayName.kt

```diff
@@ -0,0 +1,11 @@
+package com.gravatar.ui.components.atomic
+
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.gravatar.api.models.UserProfile
+
+@Composable
+public fun DisplayName(profile: UserProfile, modifier: Modifier = Modifier) {
+    Text(text = profile.displayName ?: "", modifier = modifier)
```

Comment:
I know this is kind of useless and limiting, but there is one good thing about all these components: they take a UserProfile and nothing else. Also this would help someone who just need to know what's the logic (in this case using the "displayName" field or the empty string).

Reply:
To me, it makes complete sense.

⛏
```suggestion
    Text(text = profile.displayName.orEmpty(), modifier = modifier)
```

Reply:
fixed in 069e0eae065a0da28fc1a244f1c49788b2002ec0